frontend/routes/Wifi.jsx

```diff
@@ -10,44 +28,268 @@ export class Component extends React.Component {
+            >
+              {headCell.label}
+              {orderBy === headCell.id ? (
+                <span className={classes.visuallyHidden}>
+                  {order === 'desc' ? 'sorted descending' : 'sorted ascending'}
+                </span>
+              ) : null}
+            </TableSortLabel>
+          </TableCell>
         ))}
-      </tbody>
-    </table>
+      </TableRow>
+    </TableHead>
   );
 }
+
+EnhancedTableHead.propTypes = {
+  classes: PropTypes.object.isRequired,
+  onRequestSort: PropTypes.func.isRequired,
+  order: PropTypes.oneOf(['asc', 'desc']).isRequired,
+  orderBy: PropTypes.string.isRequired,
+};
+
+const useToolbarStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2),
+  },
+  highlight:
+    theme.palette.type === 'light'
+      ? {
+          color: theme.palette.secondary.main,
+          backgroundColor: lighten(theme.palette.secondary.light, 0.85),
+        }
+      : {
+          color: theme.palette.text.primary,
+          backgroundColor: theme.palette.secondary.dark,
+        },
+  title: {
+    flex: '1 1 100%',
+  },
+}));
+
+const EnhancedTableToolbar = (props) => {
+  const classes = useToolbarStyles();
+  return (
+    <Toolbar>
+        <Typography className={classes.title} variant="h6" id="tableTitle" component="div">
+          Wifi 
+        </Typography>
+
+    </Toolbar>
+  );
+};
+
+EnhancedTableToolbar.propTypes = {
+};
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '90%',
+    margin: theme.spacing(2),
+  },
+  paper: {
+    width: '100%',
+    padding: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+    
+  },
+  table: {
+    minWidth: 550,
+  },
+  visuallyHidden: {
+    border: 0,
+    clip: 'rect(0 0 0 0)',
+    height: 1,
+    margin: -1,
+    overflow: 'hidden',
+    padding: 0,
+    position: 'absolute',
+    top: 20,
+    width: 1,
+    
+  },
+}));
+
+function descendingComparator(a, b, orderBy) {
+  let is_num = false;
+  headCells.map((element)=>{
+    if(element.id == orderBy)
+      is_num = is_num || element.numeric
+  });
+
+  if(is_num){
+    if (parseInt(b[orderBy]) < parseInt(a[orderBy])) {
+      return -1;
+    }
+    if (parseInt(b[orderBy]) > parseInt(a[orderBy])) {
+      return 1;
+    }
+    return 0;
+  }
+  else{
+    return a[orderBy] < b[orderBy] ? -1 : a[orderBy] > b[orderBy];    
+  }
+
+
+}
+
+function getComparator(order, orderBy) {
+  
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+function stableSort(array, comparator) {
+  const stabilizedThis = array.map((el, index) => [el, index]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) return order;
+    return a[1] - b[1];
+  });
+  return stabilizedThis.map((el) => el[0]);
+}
+
+function EnhancedTable(props) {
+  const classes = useStyles();
+  const [order, setOrder] = React.useState('asc');
```

Comment:
To keep consistent with the rest of the codebase please import `useState` directly.